app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
+{
+    public function index(){
```

Comment:
```suggestion
    public function index(): View
    {
```
Don't forget about being explicit about a method's return type.